packages/backup-restore-operator/charts/templates/rancher-resourceset.yaml

```diff
@@ -17,16 +17,32 @@ resourceSelectors:
   - apiVersion: "rbac.authorization.k8s.io/v1"
     kindsRegexp: "^clusterrolebindings$"
     resourceNameRegexp: "^cattle-|^clusterrolebinding-|^globaladmin-user-|^grb-u-"
+    resourceNames:
+      - "eks-operator"
   - apiVersion: "rbac.authorization.k8s.io/v1"
     kindsRegexp: "^clusterroles$"
     resourceNameRegexp: "^cattle-|^p-|^c-|^local-|^user-|^u-|^project-|^create-ns$"
+    resourceNames:
+      - "eks-operator"
```

Comment:
So if it matches either `resourceNameRegexp` or `resourceNames` it catches it ?

Reply:
yes, it's an OR, `resourceNameRegexp || resourceNames`